fennel/datasets/datasets.py

```diff
@@ -169,24 +170,6 @@ def field(
 # ---------------------------------------------------------------------
 
 
-def _validate_join_bounds(within: Tuple[Duration, Duration]):
```

Comment:
Why delete this ?

Reply:
This was being called from Join class although we have the same code in visitJoin. Removing to reduce code duplication.